devicon.json

```diff
@@ -3776,6 +3776,34 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "glitch",
+        "altnames": [],
+        "tags": [
+            "web-apps",
+            "online-platform",
+            "programming-environment",
+            "collaborative-environment",
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain",
+            ],
+            "font": []
+        },
+        "color": "#000000",
+        "aliases": [
```

Comment:
The original and plain SVGs do not include the word "glitch". Therefore, they cannot serve as aliases for the wordmark SVGs.